.github/ISSUE_TEMPLATE/5_release.yml

```diff
@@ -58,7 +58,7 @@ body:
       description: |
         Changes introduced by this release.
         List links to the PRs associated to every commit in the release. This will automatically create references between PRs and this release that will be very valuable for traceability.
-        TIP: use the autogenerate changelog feature during the draft release.
+        TIP: use the autogenerate changelog feature during the draft release. (TIP: use regex replace for better visualization ``\*.+ by (@\w+) in (https:.*)`` -> ``* $2 by $1``)

```

Comment:
use regex replace. how so? is this in bash?

Reply:
No that is just a tip that i wanted to document somewhere for myself, when you are creating a release issue, and you copy&paste there the change log from the draft tag, it doesn't looks nice as the information there is duplicated. This is just a helper how to make it nicer.